README.md

```diff
@@ -22,4 +24,13 @@ Contact the IRIDA Team
 ----------------------
 * To ask general or usage questions, email the IRIDA team at IRIDA-mail@sfu.ca
 * Report bugs, suggest features, or contribute code on [GitHub](http://github.com/phac-nml/irida)
-* Chat with the developers on [Gitter](https://gitter.im/irida-project/)
\ No newline at end of file
+* Chat with the developers on [Gitter](https://gitter.im/irida-project/)
+
+Citation
+--------
+
+A paper describing IRIDA is available on bioRxiv:
```

Comment:
Can you add something about that the IRIDA pub is in preparation and this is a preprint?

Reply:
Yep. Done in 3244721e50